arches/install/arches-templates/tests/run_tests.py

```diff
@@ -0,0 +1,39 @@
+"""
+ARCHES - a program developed to inventory and manage immovable cultural heritage.
+Copyright (C) 2013 J. Paul Getty Trust and World Monuments Fund
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+
+import sys
+import os
+from django.test.utils import get_runner
+from tests import test_settings
+
+
+def run_all(argv=None):
```

Comment:
The CI workflow in this PR just uses `manage.py test` like AfS does, so this file isn't even used. Shall we remove it? (And from core, too?)

Reply:
I don't think it's cruft in Arches core as it's used in `setup.py`, but happy to pull it out of project templates 👍